handler/hello.go

```diff
@@ -0,0 +1,65 @@
+package handler
+
+import (
+	"net/http"
+	"encoding/json"
+	"fmt"
+
+	"github.com/ONSdigital/dp-cantabular-dimension-api/contract"
+	"github.com/ONSdigital/dp-api-clients-go/v2/cantabular"
+)
+
+// Hello handles requests to /hello
+type Hello struct{
+	respond responder
+	ctblr   cantabularClient
+}
+
+// NewHello returns a new Hello handler
+func NewHello(r responder, c cantabularClient) *Hello {
+	return &Hello{
+		respond: r,
+		ctblr:   c,
+	}
+}
+
+// Get is the handler for GET /hello
+func(h *Hello) Get(w http.ResponseWriter, r *http.Request){
+	ctx := r.Context()
+
+	resp := contract.GetHelloResponse{
+		Message: "Hello, World!",
+	}
+
+	h.respond.JSON(ctx, w, http.StatusOK, resp)
+}
+
+// Post is the handler for POST /hello - Is used for an error example
```

Comment:
Change Post to Create: 
```suggestion
// Create is the handler for POST /hello - Is used for an error example
```